heartbeat/azure-lb

```diff
@@ -15,11 +15,17 @@
 # Initialization:
 : ${OCF_FUNCTIONS_DIR=${OCF_ROOT}/lib/heartbeat}
 . ${OCF_FUNCTIONS_DIR}/ocf-shellfuncs
-
+. ${OCF_FUNCTIONS_DIR}/ocf-distro
 # Defaults
-OCF_RESKEY_nc_default="/usr/bin/nc"
 OCF_RESKEY_port_default="61000"
 
+# On SLES we use socat
+if [ -e /usr/bin/socat ]; then
```

Comment:
@varkoly as usually both are installed by default, it will always use `socat`, right? Also, there is a OCF function that already tests the binary existence. There is no sense to duplicate that. https://github.com/ClusterLabs/resource-agents/blob/master/heartbeat/azure-lb#L164

More than that, if the binary is installed in different location, the RA will not work.

I would remove that check, and if the user does not inform socat, nc will be used by default.

Reply:
+1 regarding letting it use nc by default.